test/system/server_tests.rb

```diff
@@ -320,7 +320,11 @@ class WithEnvIpAndPortTests < SystemTests
     setup do
       # get our current ip address, need something different than 0.0.0.0 and
       # 127.0.0.1 to bind to
-      ENV['SANFORD_IP']   = IPSocket.getaddress(Socket.gethostname)
+      ENV['SANFORD_IP'] = Socket.getaddrinfo(
+        Socket.gethostname,
+        80,
+        Socket::AF_INET
+      ).first[3]
```

Comment:
@kellyredding - Meant to mention this in the commit, the `IPSocket.getaddress(Socket.gethostname)` started returning my IPv6 address which caused the tests to fail. 

Using `Socket.getaddrinfo` and passing `Socket::AF_INET` ensures we get a list of the current IPv4 addresses. The `80` is a "service" (http) or port number and in this case doesn't effect what we are trying to get. It is required though so I chose something random.